app/user/invite-users/invite-users.component.html

```diff
@@ -0,0 +1,29 @@
+<md-card class="default-card">
+  <h1>{{ 'invitation' | translate}}</h1>
+</md-card>
+<md-card class="default-card">
+  <div>
+    <md-hint class="error-hint">
+      <span *ngIf = "isError">
+        {{'Please remove following emails' | translate}}:
+        <span *ngFor = "let errorItem of errorItems">
+          <div>{{errorItem}}</div>
+        </span>
+      </span>
+    </md-hint>
+    <tag-input [ngModel]="emails"
+               separatorKeys="[32]"
+               (onRemove)="onRemove($event)"
+            	 (onAdd)="onAdd($event)"
+               [transform]="validateEmail"
+               [secondaryPlaceholder] ="('Enter email ids' | translate)"
+               [placeholder]="('+ Email')">
+    </tag-input>
```

Comment:
the new version of TagInput is out and mess up the design a little bit. just run `npm i ` to get the new version of TagInput. You can look into `hip-user-tag-input` to get an idea how to do it.